components/lib/calendar/Calendar.d.ts

```diff
@@ -532,10 +532,15 @@ export interface CalendarProps {
      */
     showIcon?: boolean | undefined;
     /**
-     * Icon position of the component. This only applies if the showIcon option is set to true.
+     * Visual style for the icon. This only applies if the showIcon option is set to true.
```

Comment:
(1) The documentation seemed misleading as talk about the icon's position instead of the display style.